src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -66,10 +79,32 @@ class VisitService(
 
     // check if the existing visit is in the past
     validateVisitStartDate(visit, "changed")
-    return reserveVisitSlot(bookingReference, reserveVisitSlotDto)
+    val visitDto = createApplication(bookingReference, reserveVisitSlotDto)
+
+    saveEventAudit(
+      reserveVisitSlotDto.actionedBy,
+      visitDto,
+      if (visitDto.visitStatus == RESERVED) RESERVED_VISIT else CHANGING_VISIT,
+      NOT_KNOWN,
```

Comment:
will it be NOT_KNOWN for reserved visits and when the visit is booked we set the application type. is that right?